### ISSUE TITLE
release to crates-io

### ISSUE THREAD

**Issue by flxzt** [tfachmann/typst-as-library#3]
Hi, thanks for writing this wrapper.
It is much more conventient to use than having to write and maintain the typst world boilerplate code myself.

Are you planning to publish this to crates-io, possibly syncing releases with typst? This way it can be ensured that the sources will always be available. :)

**Comment by tfachmann** [comment]
You're welcome!
This was more of a case study for me, unifying the available resources.
At the time of writing this boilerplate, `typst` wasn't event released to crates.io and I needed a git reference to it to function (and therefore making it impossible to uploada to crates.io)

Ideally, typst itself provides a more intuitive API to work with.

But until then, it may be beneficial for others to upload it, what do you think?

**Comment by flxzt** [comment]
Yes, I think until typst has improved their `World` API it would be nice to use this wrapper as a published crate. :) I think I have a fix for #2 and will open a PR for it soon, what other issues need to be resolved/features implemented before doing a release in your opinion?

**Comment by tfachmann** [comment]
I think with that, we could already create a release!

**Comment by tfachmann** [comment]
Please have a look at a successor of this library: https://github.com/Relacibo/typst-as-lib
It is also released on crates.io as [typst-as-lib](https://crates.io/crates/typst-as-lib) @Relacibo.

I will keep this repository here as a simple, minimal _example_ how to make calls to the typst library from your code to get inspiration from and adapt to your needs.
Note that there was little to no examples out there when I first created this repository.

If you want to `cargo add` something, look at the linked crate.
